packages/core/src/oidc/grants/refresh-token.ts

```diff
@@ -93,8 +90,6 @@ export const buildHandler: (
     },
   } = providerInstance.configuration();
 
-  const dPoP = await dpopValidate(ctx);
```

Comment:
jc: should we check the dPoP here instead? does the order matter?

Reply:
The order does not matter, and all the existing tests passed. This change makes the code easier to read.